packages/cubejs-testing/src/REQUIRED_ENV_VARS.ts

```diff
@@ -19,7 +19,7 @@ export const REQUIRED_ENV_VARS: {[key: string]: string[]} = {
     'CUBEJS_DB_USER',
     'CUBEJS_DB_PASS',
     'CUBEJS_DB_NAME',
-    'CUBEJS_FIREBOLT_ENGINE_ENDPOINT'
+    'CUBEJS_FIREBOLT_ENGINE_NAME'
```

Comment:
Add `CUBEJS_FIREBOLT_ACCOUNT` here too?

Reply:
`CUBEJS_FIREBOLT_ACCOUNT` is not really required. Firebolt sdk resolves yours default account if account is not specified. In Firebolt your user may be a member of > 1 account, only then you need to specify account name.

Reply:
Fair enough. That is a good point!

For background, the naming here is not that fortunate, the goal of these variables is two fold:
1. Ensure that the auth is correctly set.
2. Copy auth vars into Docker environment, if using Docker-ized cubejs.
I will make a separate PR to automatize 2. based on env var prefixes, so it gets less confusing.